Examples/Tutorials/Tutorial-LiveTraffic/main.cpp

```diff
@@ -185,12 +184,8 @@ int main(int argc, char* argv[])
 	dev->stopCapture();
 
 	// go over the packet vector and feed all packets to the stats object
-	for (pcpp::RawPacketVector::ConstVectorIterator iter = packetVec.begin(); iter != packetVec.end(); iter++)
-	{
-		// parse raw packet
-		pcpp::Packet parsedPacket(*iter);
-
-		// feed packet to the stats object
+	for (const auto& packet : packetVec) {
```

Comment:
nit: I'm not sure but library mostly uses `{` in newline instead of end of the line.

Reply:
Thanks @egecetin for pointing this out. I've reviewed the codebase and adjusted the brace style to ensure consistency with the existing code. The updated code is now committed. Please let me know if further adjustments are needed.